qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -1206,3 +1207,128 @@ def new_modulated_gaussian_control(
         detunings=np.zeros(segment_count),
         durations=np.array([segment_duration] * segment_count),
     )
+
+
+def new_drag_control(
+    rabi_rotation: float,
+    segment_count: int,
+    duration: float,
+    width: float,
+    beta: float,
+    name: Optional[str] = None,
+) -> DrivenControl:
+    r"""
+    Generates a Gaussian driven control sequence with DRAG correction applied.
+
+    The addition of DRAG further reduces leakage out of the qubit subspace via an additional
+    off-quadrature corrective driving term proportional to the derivative of the Gaussian pulse.
+
+    Parameters
+    ----------
+    rabi_rotation : float
+        Total Rabi rotation :math:`\theta` to be performed by the driven control.
+    segment_count : int
+        Number of segments in the control sequence.
+    duration : float
+        Total duration :math:`t_g` of the control sequence.
+    width : float
+        Width (standard deviation) :math:`\sigma` of the ideal Gaussian pulse.
+    beta : float
```

Comment:
FWIW, I wasn't sure about this parameter before (why can't we use \delta here as in the paper?), which would just because of my limited knowledge on DRAG. Then I had a look at the API from IBM
https://qiskit.org/documentation/stubs/qiskit.pulse.Drag.html
which uses the same notation. I wonder whether this is a convention for DRAG?

Reply:
From reading the literature, my impression was that this notation is fairly standard (see for example this experimental result https://journals.aps.org/pra/pdf/10.1103/PhysRevA.82.040305). I think one point of confusion is that depending on your system, \lambda_1 can be taken to be 1 which might change your choice of notation. You're right that Qiskit uses \beta too but interestingly enough, when I had a look at Qiskit's implementation I think I also found a fairly significant bug in theirs.

Reply:
Haha, nice. I am happy with the name.